localenv/happy-life-bank/docker-compose.yml

```diff
@@ -32,6 +32,38 @@ services:
       interval: 30s
       retries: 1
       timeout: 3s
+  happy-life-bank-point-of-sale: 
+    hostname: happy-life-bank-point-of-sale
+    image: rafiki-point-of-sale 
+    build: 
+      context: ../.. 
+      dockerfile: ./packages/point-of-sale/Dockerfile.dev 
+    restart: always 
+    networks: 
+      - rafiki 
+    ports: 
+      - '3007:3007' 
+    volumes: 
+      - type: bind 
+        source: ../../packages/point-of-sale/src 
+        target: /home/rafiki/packages/point-of-sale/src 
+        read_only: true 
+    environment: 
+      NODE_ENV: ${NODE_ENV:-development} 
+      INSTANCE_NAME: HAPPY-LIFE 
+      TRUST_PROXY: ${TRUST_PROXY} 
+      LOG_LEVEL: debug 
+      CARD_SERVICE_PORT: 3007 
```

Comment:
Need to change the env variable name to match the pos service's config, as well as update the port number so it doesn't conflict with the card service.